composer.json

```diff
@@ -13,23 +13,34 @@
         }
     ],
     "require": {
-        "sonata-project/media-bundle": "~3.0",
-        "sonata-project/admin-bundle": "~3.40"
+        "php": "^7.2",
+        "symfony/dependency-injection": "^3.4 || ^4.0",
+        "symfony/http-foundation": "^3.4 || ^4.0",
+        "symfony/http-kernel": "^3.4 || ^4.0",
+        "symfony/security-core": "^3.4 || ^4.0",
+        "symfony/config": "^3.4 || ^4.0",
+        "sonata-project/media-bundle": "^3.0",
+        "sonata-project/admin-bundle": "^3.40"
     },
     "require-dev": {
-        "symfony/phpunit-bridge":       "~2.7|~3.0",
-        "phpunit/phpunit":              "~4.8|~5.0"
+        "symfony/phpunit-bridge": "^3.4.19 || ^4.0"
```

Comment:
TIL before 3.4.19 Symfony didn't use composer to download PHPUnit, but it used to download a zip directly from GitHub and extract it, however, this doesn't work anymore since GitHub changed the URL endpoint for downloading releases, hence why we need ^3.4.19